.github/workflows/deploy_preview.yml

```diff
@@ -30,20 +30,20 @@ jobs:
     steps:
       - name: Checkout source
         uses: actions/checkout@v4
-      
+
       - name: .NET Setup SDKs
         uses: actions/setup-dotnet@v4
         with:
           dotnet-version: |
             8.0.x
-          include-prerelease: true
-     
-      - name: .NET Builld 
-        run: dotnet build ${{ env.PROJECT }} -c ${{ env.BUILD_CONFIG }} -o publish -f net8.0 -r linux-x64 --self-contained=true -p:BuildNumber=$BUILD_NUMBER -p:SourceRevisionId=$GITHUB_SHA -p:ContinuousIntegrationBuild=true
-      
-      - name: .NET Publish 
-        run: dotnet publish ${{ env.PROJECT }} -c ${{ env.BUILD_CONFIG }} -o publish -f net8.0 -r linux-x64 --self-contained=true -p:BuildNumber=$BUILD_NUMBER -p:SourceRevisionId=$GITHUB_SHA -p:ContinuousIntegrationBuild=true
-       
+            9.0.x
+
+      #- name: .NET Builld
```

Comment:
Although the Publish will do the Build if it doesn't exist. I think it's better to keep a Build and then make a Publish (the bin will already be present, so it won't do this compilation again).